fe/src/main/java/org/apache/doris/task/ClearTransactionTask.java

```diff
@@ -17,41 +17,29 @@
 
 package org.apache.doris.task;
 
-import java.util.List;
-
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
 import org.apache.doris.thrift.TClearTransactionTaskRequest;
 import org.apache.doris.thrift.TTaskType;
 
-public class ClearTransactionTask extends AgentTask {
+import java.util.List;
 
-    private static final Logger LOG = LogManager.getLogger(ClearTransactionTask.class);
+public class ClearTransactionTask extends AgentTask {
 
     private long transactionId;
     private List<Long> partitionIds;
-    private boolean isFinished;
 
-    public ClearTransactionTask(long backendId, long transactionId, 
-            List<Long> partitionIds) {
+    public ClearTransactionTask(long backendId, long transactionId, List<Long> partitionIds) {
         super(null, backendId, TTaskType.CLEAR_TRANSACTION_TASK, -1L, -1L, -1L, -1L, -1L, transactionId);
         this.transactionId = transactionId;
         this.partitionIds = partitionIds;
         this.isFinished = false;
     }
     
     public TClearTransactionTaskRequest toThrift() {
-        TClearTransactionTaskRequest clearTransactionTaskRequest = new TClearTransactionTaskRequest(transactionId, 
-                partitionIds);
+        TClearTransactionTaskRequest clearTransactionTaskRequest = new TClearTransactionTaskRequest();
+        clearTransactionTaskRequest.setTransaction_id(transactionId);
+        if (partitionIds != null && !partitionIds.isEmpty()) {
```

Comment:
partitionIds in TClearTransactionTaskRequest is required, if you don't set it, it will cause serialization problem.

Reply:
Forget to change it. I will fix it